types/span.go

```diff
@@ -35,31 +34,24 @@ type CoreSpanMetadata struct {
 	DurationMs   float64 `json:"durationMs,omitempty"`
 }
 
-// convertTimestamp turns a Zipkin timestamp (a Unix timestamp in microseconds)
+type annotation struct {
```

Comment:
Create a new annotation/endpoint used by the honeycomb data structure, to split it from the annotation/endpoint structs used by zipkin.

This way in case we need to add more api versions, or if we need to update the zipkin api, we don't need to change the honeycomb interface.